### PR TITLE
Docs minor improvements

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
The "simple example" being the entry gate into SSM.jl (i.e., users will try that first and if they understand everything they will continue and adopt the package), it's important to make it as clear and streamlined as possible ☺️

### CODE REVIEW

File: docs/src/turing_simple.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-## Simple Example
 
-### Load Packages
+## Load Packages
 
 The first step is to load the required packages. You will need to install each package in your local
 environment in order to run the code locally. We will also set a random number generator so that the results are reproducible.
 
-```@setup Turing
+```@setup turing_simple
```
Comment (DominiqueMakowski) [inline docs/src/turing_simple.md:14]: It seems like this chunk is not showing, is it because it's a setup chunk?

**Comment by DominiqueMakowski** [comment]
Also, could you add the documentation URL to the "website" field of the *About* repo section, some people (like me 😁) get disoriented when it's missing 

image

File: docs/src/turing_simple.md [hunk 2]
```diff
@@ -21,22 +21,24 @@ using StatsPlots
...
 
-```@example Turing
+```@example turing_simple
 # Generate some data with known parameters
 dist = LBA(ν=[3.0, 2.0], A = .8, k = .2, τ = .3)
 data = rand(dist, 100)
 ```
 
-### Specify Turing Model
+The `rand()` function will sample random draws from the distribution, and store that into a tuple of 2 vectors (one for `choice` and one for `rt`).
```
Comment (itsdfish) [inline docs/src/turing_simple.md:34]: Uses a NamedTuple here.

**Comment by itsdfish** [comment]
> Also, could you add the documentation URL to the "website" field of the _About_ repo section, some people (like me grin) get disoriented when it's missing
> 
> image

Can do.

Thanks for the PR. Please see my comments.

**Comment by DominiqueMakowski** [comment]
> Please see my comments.

I think all have been addressed?

**Comment by itsdfish** [comment]
@DominiqueMakowski, thanks for addressing the comments. I will merge now.

**Pull request merged**